ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -250,7 +251,8 @@ protected void onAttachedToWindow() {
    */
   @Override
   public void requestChildFocus(View child, View focused) {
-    if (focused != null) {
+    boolean isWebView = focused instanceof WebView;
+    if (focused != null && !isWebView) {
```

Comment:
This could cause issues and completely deny scrollTo a WebView.. It is not clear to me the expected behaviour for this functionality, but I decided to publish the pull request and listen to the community feedback. Thanks